packages/boxed-expression-component/src/selection/BeeTableSelectionContext.tsx

```diff
@@ -699,12 +700,19 @@ export function BeeTableSelectionContextProvider({ children }: React.PropsWithCh
 
           // Selecting a header cell from another header cell
           // Do not allow selecting multi-line header cells
-          if (
-            (prev.selectionEnd?.rowIndex ?? 0) < 0 &&
-            (newSelectionEnd?.rowIndex ?? 0) < 0 &&
-            prev.selectionEnd?.rowIndex !== newSelectionEnd?.rowIndex
-          ) {
-            return prev;
+          if ((prev.selectionEnd?.rowIndex ?? 0) < 0 && (newSelectionEnd?.rowIndex ?? 0) < 0) {
+            if (prev.selectionEnd?.rowIndex !== newSelectionEnd?.rowIndex) {
```

Comment:
@jomarko I'm not familiar with this logic, can you please clarify when a rowIndex is negative? Are the headers managed with negative indexes?

Reply:
Yes, negative indexes are used for header rows.

Lets say we have a decision table with two header rows. The `top` header row has index `-2`, the `bottom` header row has an index `-1`.

The `first data` row has index `0`.